_release/bat/release_test.py

```diff
@@ -405,10 +405,6 @@ def test_start_all_workloads(self):
         """Start one instance of all workloads"""
         self.failUnless(launch_all_workloads())
 
-    def test_start_all_workloads10(self):
```

Comment:
My comment on this from the prior PR is lost since it was closed.  I've opened https://github.com/01org/ciao/issues/749 to track re-adding a test case to launch a herd of cattle in BAT.